packages/cfpb-forms/src/atoms/label.less

```diff
@@ -30,5 +30,13 @@
                 margin-top: unit( 10px / @base-font-size-px, em );
             }
         }
+
+        .respond-to-max( @bp-xs-max, {
+            @h4-font-size-on-xs: @base-font-size-px;
```

Comment:
Why is defining a new variable `@h4-font-size-on-xs` necessary?

Reply:
The other definition is scoped the the `.respond-to-max`, so this one needs to be as well. I could pull the `@h4-font-size-on-xs` definition out (or just use `base-font-size-px` directly) instead. Thoughts?

Reply:
I think the original reason for the defining the variable was to make a point that, instead of stepping down one heading level (like h1-h4) we're just defining a different font size.

Reply:
Are you talking about the variable defined [here](https://github.com/cfpb/design-system/blob/d67f9b6af2ee320ef0baed1e97e904c38c8ea9a9/packages/cfpb-core/src/base.less#L65)? I was thinking just use `@base-font-size-px` directly. Seems like a question for @Scotchester

Reply:
How about just changing the mixin that `.a-label__heading` includes to `.h4()` and then it will inherit the media query?

Reply:
Aha! Updated in d678fe4a